Remotion/Data/DomainObjects.UnitTests/Serialization/DataContainerTest.cs

```diff
@@ -1,170 +0,0 @@
-// This file is part of the re-motion Core Framework (www.re-motion.org)
-// Copyright (c) rubicon IT GmbH, www.rubicon.eu
-// 
-// The re-motion Core Framework is free software; you can redistribute it 
-// and/or modify it under the terms of the GNU Lesser General Public License 
-// as published by the Free Software Foundation; either version 2.1 of the 
-// License, or (at your option) any later version.
-// 
-// re-motion is distributed in the hope that it will be useful, 
-// but WITHOUT ANY WARRANTY; without even the implied warranty of 
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
-// GNU Lesser General Public License for more details.
-// 
-// You should have received a copy of the GNU Lesser General Public License
-// along with re-motion; if not, see http://www.gnu.org/licenses.
-// 
-using System;
-using System.Runtime.Serialization;
-using NUnit.Framework;
-using Remotion.Data.DomainObjects.DataManagement;
-using Remotion.Data.DomainObjects.UnitTests.TestDomain;
-using Remotion.Development.NUnit.UnitTesting;
-using Remotion.Development.UnitTesting;
-
-namespace Remotion.Data.DomainObjects.UnitTests.Serialization
```

Comment:
I am not sure that we want to delete all of these files in their entirety. Many files also do `[Serializable]` checks, which we might still want? Other files do additional tests that are based of `[Serializable]` so they should probably stay as well?

Paging @MichaelKetting to answer this